tjpcov/covariance_fourier_ssc.py

```diff
@@ -142,22 +173,7 @@ def get_covariance_block(
         )
 
         masks = self.get_masks_dict(tr, {})
```

Comment:
In the case of fsky, I don't think you want to call this function. I would move this line to `_get_sigma2_B` and pass instead the tracer dictionary